app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -181,7 +181,6 @@ def create_group_assignment_repo(selected_group: group, new_group_title: nil)
       selected_group,
       new_group_title
     )
-
```

Comment:
This delta probably shouldn't be in this PR.

Reply:
I'll fix this 👍